package.json

```diff
@@ -35,12 +35,8 @@
   "scripts": {
     "test:unit": "yarn run test:tooling && yarn run test:cc-widgets",
     "test:tooling": "jest --coverage",
-    "test:cc-widgets": "yarn run test:store && yarn run test:widgets",
-    "test:store": "yarn workspace @webex/cc-store test:unit",
-    "test:widgets": "yarn workspace @webex/cc-station-login test:unit && yarn workspace @webex/cc-user-state test:unit",
-    "build": "yarn run build:store && yarn run build:widgets",
-    "build:store": "yarn workspace @webex/cc-store build:src",
-    "build:widgets": "yarn workspace @webex/cc-station-login build:src && yarn workspace @webex/cc-user-state build:src",
+    "test:cc-widgets": "yarn workspaces foreach --all --parallel --include @webex/cc-store --include @webex/cc-station-login --include @webex/cc-user-state  run build:src",
+    "build": "yarn workspaces foreach --all --topological --parallel --include @webex/cc-store --include @webex/cc-widgets --include @webex/cc-station-login --include @webex/cc-user-state  run build:src",
```

Comment:
Let us exclude the samples and let for each handle all the other packages

Reply:
Done